src/containers/Home/Home.css

```diff
@@ -1,28 +1,114 @@
-.home-container {
+/* .home-container {
   display: flex;
   flex-wrap: wrap;
   width: 100%;
   justify-content: center;
+} */
+.home-container {
+  display: flex;
+  /* flex-direction: column; */
+  background-image: linear-gradient(
+      rgba(6, 18, 90, 0.2),
+      rgba(14, 44, 99, 0.2)
+    ),
+    url("https://texturify.com/content/10769/Sky%20Background-56.jpg");
+  background-position: center center;
+  background-attachment: fixed;
+  background-size: cover;
+  height: auto;
+  min-height: 90vh;
+  width: 100%;
+  padding: 50px 0;
 }
 
+.home-container img {
+  height: 1.5em;
+  margin-top: 0.3em;
+}
+
+.header-title {
+  display: flex;
+  align-items: center;
+}
 .address-form {
+  background-color: whitesmoke;
+  background-image: linear-gradient(#6B9AC4, #EFF2F1);
+  opacity: 0.82;
   display: flex;
   flex-direction: column;
   border: 2px solid lightblue;
   border-radius: 10px;
   width: 40%;
   align-items: center;
-  margin-top: 15%;
+  margin: auto;
+  /* margin-top: 15%; */
   padding-bottom: 0%;
-  padding: 5%;
+  padding: 4%;
+  border-radius: 7px;
+  box-shadow: 10px 10px 10px rgba(0, 0, 0, 0.05);
 }
 
 .address-form input {
-  width: 75%;
-  text-align: center;
-  margin-bottom: 5%;
+  opacity: 1;
```

Comment:
I'm sure the configure component will be much easier and faster to style compared to all the troubles these inputs gave us.